tests/common/dualtor/dual_tor_io.py

```diff
@@ -51,22 +52,24 @@ def __init__(self, activehost, standbyhost, ptfhost, ptfadapter, tbinfo,
 
         self.time_to_listen = 180.0
         self.sniff_time_incr = 60
-        self.send_interval = 0.0035 # Inter-packet interval
+        # Inter-packet send-interval (minimum interval 3.5ms)
+        self.send_interval = send_interval if send_interval and send_interval > 0.0035 else 0.0035
```

Comment:
Can we add a warning log here in case the user passes a values < .0035?

Reply:
Sure. Added now.